x-pack/plugins/apm/server/lib/metrics/by_agent/java/thread_count/index.ts

```diff
@@ -57,6 +57,6 @@ export async function getThreadCountChart(
       threadCount: { avg: { field: METRIC_JAVA_THREAD_COUNT } },
       threadCountMax: { max: { field: METRIC_JAVA_THREAD_COUNT } }
     },
-    additionalFilters: [{ term: { [SERVICE_AGENT_NAME]: 'java' } }]
+    additionalFilters: [{ term: { [AGENT_NAME]: 'java' } }]
```

Comment:
Thanks! 
fyi @dgieselaar in case you are also making some changes to `AGENT_NAME`